core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:768px) {
+/* Disabling navbar-collapse */
+  .oppia-navbar-breadcrumb>li, .oppia-navbar-breadcrumb {
+    float: left;
+  }
+  .oppia-navbar-breadcrumb {
+    margin-top: 0;
+    margin-left: 5px;
+  }
+  .navbar-nav.oppia-navbar-nav > li > a{
+    display: none;
+  }
+}
+@media(max-width:600px) {
```

Comment:
@seanlip  At around 768px we're converting to oppia icon. And get filled up with 40 character title at 600px and 450px. I chose this value as thought it was optimal and  i intially tested with 3 different devices of screen size (3.5, 4.7,5.5) inch.